tutorial-create-custom-workflows/index.html

```diff
@@ -42,16 +42,16 @@ <h2 is-upgraded>Pre-Requisites</h2>
 <ul>
 <li>Kubernetes 1.17 or later. Recommended sizing of at least: 2 vCPUs, 8GB memory, 10GB of disk space (for persistent volume creation)</li>
 <li>Helm3 or Kubectl</li>
-<li>Access to the Internet from Cluster (For Air-Gapped Environments refer to<a href="https://docs.litmuschaos.io/docs/user-guides/air-gapped/" target="_blank"> this guide</a>)</li>
```

Comment:
Is this link mentioned someplace else? What is the reason we are not mentioning Air-Gapped Environments?